driver/bpf/fillers.h

```diff
@@ -2090,6 +2092,25 @@ FILLER(proc_startupdate_3, true)
 		res = bpf_val_to_ring_type(data, loginuid.val, PT_INT32);
 		if (res != PPM_SUCCESS)
 			return res;
+
+		/*
+		 * is_exe_writable
+		 */
+
+		exe_file = _READ(mm->exe_file);
+
+		if (exe_file) {
+			struct inode *inode = _READ(exe_file->f_inode);
+			if (inode) {
+				// XXX if I remove the following two lines this compiles and runs under clang-7
+				is_exe_writable |= (inode_permission(inode, MAY_WRITE) == 0);
```

Comment:
@luca-sd - as discussed privately we can't do this kind of call in this context. I'll be happy to help in getting that rewritten for eBPF or for another review once that is done.